docs/notification_system.md

```diff
@@ -43,179 +48,188 @@ The currently defined Notification Types are:
+Each of these have their own configuration and behavioral semantics and testing them may need to be approached in different ways. The following sections will give as much detail as possible.
+
 
 ## Email
 
-The email notification type supports a wide variety of smtp servers and has support for ssl/tls connections and timeouts.
+The email notification type supports a wide variety of SMTP servers and has support for SSL/TLS connections and timeouts.
 
-### Testing considerations
+### Testing Considerations
 
 The following should be performed for good acceptance:
 
-* Test plain authentication
-* Test SSL and TLS authentication
-* Verify single and multiple recipients
+* Test plain authentication.
+* Test SSL and TLS authentication.
+* Verify single and multiple recipients.
 * Verify message subject and contents are formatted sanely.  They should be plaintext but readable.
 
 ### Test Service
 
-Either setup a local smtp mail service here are some options:
+Set up a local SMTP mail service.  Some options are listed below:
+
+* Postfix service on galaxy: https://galaxy.ansible.com/debops/postfix/
+* Mailtrap has a good free plan that should provide all of the features necessary: https://mailtrap.io/
 
-* postfix service on galaxy: https://galaxy.ansible.com/debops/postfix/
-* Mailtrap has a good free plan and should provide all of the features we need under that plan: https://mailtrap.io/
 
 ## Slack
 
-Slack is pretty easy to configure, it just needs a token which you can get from creating a bot in the integrations settings for the slack team.
+Slack is simple to configure; it requires a token, which you can get from creating a bot in the integrations settings for the Slack team.
 
-### Testing considerations
+### Testing Considerations
 
 The following should be performed for good acceptance:
 
-* Test single and multiple channels and good formatting of the message.   Note that slack notifications only contain the minimal information
+* Test single and multiple channels and good formatting of the message. Note that slack notifications only contain the minimal information.
 
 ### Test Service
 
-Any user of the Ansible slack service can create a bot integration (which is how this notification is implemented).   Remember to invite the bot to the channel first.
+Any user of the Ansible Slack service can create a bot integration (which is how this notification is implemented). Remember to invite the bot to the channel first.
 
-## Hipchat
 
-There are several ways to integrate with hipchat.  The Tower implementation uses Hipchat "Integrations".  Currently you can find this at the bottom right of the main hipchat webview.  From there you will select "Build your own Integration".   After creating that it will list the `auth_token` that needs to be supplied to Tower.  Some other relevant details on the fields accepted by Tower for the Hipchat notification type:
+## HipChat
```

Comment:
I'm fine if we go ahead and remove this entire section - HipChat is being shutdown soon:

> On July 26, 2018, Atlassian announced that HipChat and Stride would be discontinued February 15, 2019, and that it had reached a deal to sell their intellectual property to Slack.